scripts/bumper/main.go

```diff
@@ -284,11 +286,12 @@ func detectNewCommits(ctx context.Context, logger *logrus.Entry, stagingDir, cen
 		output, err := runCommand(logger, exec.CommandContext(ctx,
 			"git", "log",
 			centralRef,
-			"-n", "1",
+			"-n", strconv.Itoa(history),
```

Comment:
If `Iota` fails? Do we just get a complaint from `git`?

Reply:
```
// Itoa is equivalent to FormatInt(int64(i), 10).
func Itoa(i int) string {}
```

What failure specifically are you worried about?

Reply:
NVM... I was thinking it was `Atoi`...